changelogs/fragments/7533-add-test-fqdn_valid.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - fqdn_valid test - adds test to validates FQDN names against RFC 112 (https://github.com/ansible-collections/community.general/pull/7533).
```

Comment:
Please remove this, new tests are documented by their `version_added` + `short_description` in documentation.